src/web/documentation/documentation.ts

```diff
@@ -0,0 +1,185 @@
+import * as vscode from 'vscode';
+import {YamlUtils} from '../libs/yamlUtils';
+import Markdown from '../libs/markdown';
+import ApiClient from '../apiClient';
+import taskHome from './task_home.md';
+import basic from './basic.md';
+
+let channel = vscode.window.createOutputChannel("Kestra - Docs");
+
+const kestraBaseUrl = "https://api.kestra.io/v1";
```

Comment:
Minor thing, I'd would have used a constant to handle this url in a `constants.ts` file or something similar.